apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/OpenAPIContentProvider.java

```diff
@@ -0,0 +1,292 @@
+package gov.cms.bfd.server.war.commons;
+
+/** Open API content short and value descriptions for each BFD API endpoint. */
+public final class OpenAPIContentProvider {
+  // NOTE: As much as we'd like to use Java 17 features for string concatentation,
+  // our code formatter completely mangles MUCH of the following concatenated text
+  // strings. So we'll fal back on tried-and-true structure to avoid
+  // unpleasantness.
+
+  /** Open API content short description for /Patient's identifier parameter. */
+  public static final String PATIENT_SP_RES_ID_SHORT =
+      "The patient resource identifier (_id) to search for";
+  /** Open API content value for /Patient's identifier parameter. */
+  public static final String PATIENT_SP_RES_ID_VALUE =
+      """
+    Fetch _Patient_ data using a FHIR _IdType_ identifier; an IdType
+    represents the logical identity for a resource, or as much of that
+    identity that is known. In FHIR, every resource must have a _logical ID_ which is
+    defined by the FHIR specification as:
+
+    `<code>Any combination of upper or lower case ASCII letters ('A'..'Z', and 'a'..'z', numerals ('0'..'9'),
+    '-' and '.', with a length limit of 64 characters. (This might be an integer, an un-prefixed OID, UUID
+    or any other identifier pattern that meets these constraints.)</code>`
+
+    This class contains that logical ID, and can optionally also contain a relative or absolute URL
+    representing the resource identity; the following are all valid values for IdType, and all might
+    represent the same resource:
+      - `<code>123</code> (just a resource's ID)`
+      - `<code>Patient/123</code> (a relative identity)`
+      - `<code>http://example.com/Patient/123 (an absolute identity)</code>`""";
```

Comment:
There are a few problems with these examples. I think we should separate the description content from the examples a little more clearly, the formatting here is a bit problematic/inconsistent, and the content of the latter two examples aren't applicable/appropriate here, e.g.

```
    represent the same resource.
    
    Examples:
      - `?_id=567834`
      - `?_id=1234`
```